### ISSUE TITLE
Beaker copying module to wrong directory

### ISSUE THREAD

**Issue by petems** [puppetlabs/beaker#503]
Trying to get my module with just one fact to check shellshock

Code here: https://github.com/petems/petems-shellshock_facts

This fails, which is weird because it's so simple, so I did:

`BEAKER_destroy=no bundle exec rspec spec/acceptance/class_spec.rb`

but creates the following in `/etc/puppet`:

```
root@ubuntu-server-12042-x64:/etc/puppet/modules# tree shellshock_facts/
shellshock_facts/
|-- Gemfile
|-- Gemfile.lock
|-- lib
|   |-- facter
|   |   |-- facter
|   |   |   `-- shellshock.rb
|   |   `-- shellshock.rb
|   `-- lib
|       `-- facter
|           `-- shellshock.rb
|-- LICENSE
|-- manifests
|   |-- init.pp
|   `-- manifests
|       `-- init.pp
|-- Modulefile
|-- Rakefile
|-- README
`-- README.md

7 directories, 12 files
```

Compared to the tree from the repo itself:

```
peterso@name ~/Projects/petems-shellshock_facts (master $=) $ tree .
.
├── Gemfile
├── Gemfile.lock
├── LICENSE
├── Modulefile
├── README
├── README.md
├── Rakefile
├── lib
│   └── facter
│       └── shellshock.rb
├── manifests
│   └── init.pp
└── spec
    ├── acceptance
    │   ├── class_spec.rb
    │   └── nodesets
    │       ├── centos-64-x64-docker.yml
    │       ├── centos-64-x64.yml
    │       ├── default.yml
    │       └── ubuntu-server-12042-x64.yml
    ├── spec_helper.rb
    ├── spec_helper_acceptance.rb
    └── unit
        └── facter
            └── shellshock_facts_spec.rb

8 directories, 17 files
```

**Comment by chris-reeves** [comment]
Pull request which resolves this submitted: #608

**Comment by anodelman** [comment]
PR merged.
